acctest/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust
+
+SHELL ["/bin/bash", "-c"]
+
+RUN \
+  git clone https://github.com/oxidecomputer/omicron.git --depth 1 --branch jmcarp/prereqs-omit-sudo && \
```

Comment:
Note: I had to make some small changes to the install scripts in omicron to get this to build. If we like this approach, I'll send a PR to omicron and stop pointing to my branch.

Reply:
Oh, containers.